variables.tf

```
@@ -49,3 +49,10 @@ variable "enable_incoming_email" {
   type        = "string"
   default     = true
 }
+
+variable "custom_spf" {
```

I think it's fine to just call this variable `spf_record`
```suggestion
variable "spf_record" {
```

---

It's not an SPF record, it's the text portion only.

---

Is `spf_txt_record` more appropriate then?